packages/navstar/extra/dcos-navstar.service

```diff
@@ -26,4 +26,5 @@ ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/minuteman/dets
 ExecStartPre=/usr/bin/env modprobe dummy
 ExecStartPre=-/usr/bin/env ip link add minuteman type dummy
 ExecStartPre=/usr/bin/env ip link set minuteman up
+ExecStartPre=/usr/bin/env sh -c 'echo 0 > /proc/sys/net/bridge/bridge-nf-call-iptables'
```

Comment:
We should make this configurable from `dcos-config.yaml`. This is a bit dangerous since this also disables `ebtables`. Do we have a configuration section in DC/OS for `minuteman` that will allow us to plumb a flag to turn off iptables on bridges?

I would even say that by default we should have this on (since that is backwards compatible) and turn it on only if there is an issue (containers on the same bridge trying to talk to each other).